### PR TITLE
Add .vscode repository to full_install.sh

### PR DESCRIPTION
**Pull request by mindyls**
Include clone of ".vscode" repository by default during full install/test.  This will ensure consistent VSCode settings are enabled for GeoIPS developers.

```
> full_test.sh
...
Install log: $HOME/geoips/outdirs/logs/20230819.141531_install.log
FAILED: Missing settings repo .vscode
        Please run install script, then rerun test script. 
        $HOME/geoips/geoips/tests/integration_tests/full_install.sh
...

> full_install.sh
...
Install log: $HOME/geoips/outdirs/logs/20230819.141623_install.log
Installing settings repo .vscode .... 
  $HOME/geoips/.vscode/
SUCCESS: Cloned settings repo .vscode
...

> full_test.sh
...
Install log: $HOME/geoips/outdirs/logs/20230819.135942_install.log
SUCCESS: settings repo .vscode appears to be installed successfully
    drwxrwxr-x. 3 surratt surratt 4096 Aug 19 06:49 $HOME/geoips/.vscode
...
```

### CODE REVIEW

File: docs/source/releases/v1_11_3a0.rst
Comment (mindyls) [inline docs/source/releases/v1_11_3a0.rst:17]: NOTE: Do not merge until add_release_note_to_index has been merged to main, so this PR will be automatically updated to point to main.

**Review by cpcamacho — APPROVED** [review]

File: docs/source/releases/v1_11_3a0.rst
Comment (jsolbrig) [reply docs/source/releases/v1_11_3a0.rst:17]: Done.

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**
